hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/io/storage/HoodieSparkFileWriterFactory.java

```diff
@@ -82,7 +82,7 @@ protected HoodieFileWriter newParquetFileWriter(
         config.getLong(HoodieStorageConfig.PARQUET_MAX_FILE_SIZE),
         writeSupport.getHadoopConf(), config.getDouble(HoodieStorageConfig.PARQUET_COMPRESSION_RATIO_FRACTION),
         config.getBooleanOrDefault(HoodieStorageConfig.PARQUET_DICTIONARY_ENABLED));
-    parquetConfig.getHadoopConf().addResource(writeSupport.getHadoopConf());
+    writeSupport.getHadoopConf().addResource(writeSupport.getHadoopConf());
```

Comment:
Similar here.

Reply:
reverted this. But still not sure if it makes sense. writeSupport.getHadoopConf() is in the constructor of the parquet config. So it is still adding itself.

Reply:
@jonvex could you create a ticket on this one?  We can look into this later.